synapse/rest/client/v2_alpha/account.py

```diff
@@ -158,7 +159,64 @@ def __init__(self, hs):
             hs (synapse.server.HomeServer): server
         """
         super(PasswordResetSubmitTokenServlet, self).__init__()
-        self.hs = hs
+        self.config = hs.config
```

Comment:
can you just store the field you care about, rather than the whole config object?

(also: if you refer to it as `self.config.email.threepid_behaviour_email`, then (a) that keeps mypy/pycharm/etc happier, and (b) it's more efficient)

Reply:
Good point. I've seen the `self.config = hs.config` around a ton in our constructors. May be a good idea to go in and clean them all up!

Reply:
I've just pulled out the item we need now.

Reply:
And done the same for `/submit_token` while I was here.

Reply:
and yet, we still seem to be storing the whole config object here?

Reply:
I think maybe that's not a bad thing, but I'm a bit confused.

Reply:
Err, I think I just forgot to remove the `self.config` definition. I've corrected both of the endpoints in 990178f58 as we're now combining them.